frontend/src/components/profile/EditableProfileContent.tsx

```diff
@@ -352,6 +363,13 @@ export default function EditableProfileContent({
                   Cancel
                 </Button>
               </Grid>
+              <AbstractModal
+                isModalOpen={isModalOpen}
+                handleConfirm={handleConfirm}
+                handleCancel={handleClose}
+                title="Confirmation"
+                description={`You can't undo this action`}
```

Comment:
should this say "Would you like to undo your edits?" instead?

Reply:
So, what happens here is for when after editing, they press the cancel button which clears anything they just added. The pop up happens only for the cancel button. Pressing the save button in my opinion doesn't need a modal...

Reply:
oh ok, I was reading this wrong. Initially I thought it was telling the user that they cannot cancel their edits.